_includes/components/sidebar.html

```diff
@@ -30,7 +30,10 @@
       <ul class="nav-list">
         {%- for node in site.nav_external_links -%}
           <li class="nav-list-item external">
-            <a href="{{ node.url | absolute_url }}" class="nav-list-link external">
+            <a href="{{ node.url | absolute_url }}" class="nav-list-link external"
+            {%if node.opens_in_new_tab or (node.opens_in_new_tab == nil and site.nav_external_links_new_tab) %}
```

Comment:
Is there a reason you *also* need a `nil` check? I imagine that if the user has set the config option and has also set `opens_in_new_tab`, we should still let them open in a new tab.

```suggestion
            {% if node.opens_in_new_tab or site.nav_external_links_new_tab %}
```

Reply:
The logic is described in my post above:
```
If node.opens_in_new_tab is true, OR
If node.opens_in_new_tab is not set (is nil) AND site.nav_external_links_new_tab is true
```
Hopefully that clears up my logic. Defaults and overrides were kind of hard to think about.
I think it might be better to only have individual node settings?

Reply:
Oh, gotcha! I completely misunderstood the purpose of the per-page config: you want to be able to disable it for specific links, even though the global option is enabled. Got it! I'll resolve this conversation (and re-open a small formatting fix).